draft-add-requirements.md

```diff
@@ -126,9 +216,20 @@ Narrative required.
 | Requirement | Description |
 | R8.1 | Example requirement |
 
-### Walled garden content
+### Encrypted resolvers for local or home content {#local-content}
 
-Narrative required.
+Accessing locally-hosted content can require the use of a specific resolver. For example, captive networks
+or networks with walled-garden content like media on airplane Wi-Fi networks can rely on using a
+resolver hosted on the local network.
+
+In cases where a client is using an encrypted resolver provisioned by a network, and that encrypted resolver
+is able to resolve names of local content, this can fall into the use case described in {{local-network}}. However,
+it might be necessary to discover a local encrypted resolver along with specific domains if:
+
+- the network-provisioned encrypted resolver is not able to resolve local-only names, or
```

Comment:
Why would the network provisioned encrypted resolver not able to resolve internal domain names ?

Reply:
One example that springs to mind is the current BT broadband network. There is no encrypted resolver locally, but there is a central one (doh.bt.com). If we announce to the client "You can either use Do53 on 192.168.1.254, or DoH on https://doh.bt.com/dns-query" then clients choosing DoH will also need to be told "If you need to look up *.home.arpa or *.local then ask 192.168.1.254".

Reply:
Got it, Thanks. It will be helpful to add the above explanation.

Reply:
Done.